src/components/editContainer/__snapshots__/editContainer.unit.test.js.snap

```diff
@@ -59,63 +59,3 @@ exports[`EditContainer renders 1`] = `
   </div>
 </div>
 `;
-
-exports[`NavAppBar is consistent 1`] = `
```

Comment:
Double check that the snapshot is correct

Reply:
This is actually correct. Good catch. At a point in time, the test was renamed, but the snapshot wasn't updated. I should add a precommit hook to run the tests.